tools/sdk-ai-bots/azure-sdk-qa-bot-evaluation/run.py

```diff
@@ -294,41 +299,47 @@ def output_table(baseline_results: dict[str, Any], eval_results: list[dict[str,
             "\n\n"
         )
 
-def show_results(all_results: dict[str, Any]) -> None:
+def show_results(all_results: dict[str, Any], withBaseline: bool = True) -> None:
     """Display results in a table format."""
     for name, test_results in all_results.items():
-        baseline_results = {}
-        baselineName = f"{name.split('_')[0]}-test.json"
-        baseline_path = pathlib.Path(__file__).parent / "results" / baselineName
-
-        if baseline_path.exists():
-            with open(baseline_path, "r") as f:
-                baseline_data = json.load(f)
-                for result in baseline_data[:-1]:  # Skip summary
-                    baseline_results[result["testcase"]] = result
-                baseline_results["average_score"] = baseline_data[-1]["average_score"]
-
-        output_table(baseline_results, test_results, name)
-
-def verify_results(all_results: dict[str, Any]) -> bool:
+        baseline_results = None
+        if withBaseline:
```

Comment:
By the way, python uses snake-case style for variable name.
NOTE: case style doesn't influence runtime behavior so we could make the change in another PR.